### PR TITLE
[RW-2289][Risk=low] Move to binary of Rainforest CLI

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: api/src/dev/server/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,6 @@ RUN apk --no-cache add \
         libc6-compat \
         openssh-client \
         git \
-        make \
```
Comment (gjuggler) [inline api/src/dev/server/Dockerfile]: Can you explain this and the other removed / changed lines? They seem unrelated to the purpose of this PR. If they aren't – I'm wondering what purpose they served, or what about the Rainforest change has made them unnecessary?

Or were they just cruft to begin with?

Comment (blrubenstein) [reply api/src/dev/server/Dockerfile]: Sure! Those were added to support the ruby version of the rainforest cli, which is deprecated. Wanted to remove them as they are now unneeded dependencies.

Comment (gjuggler) [reply api/src/dev/server/Dockerfile]: Awesome – thanks for clarifying.
